bin/create-config

```diff
@@ -14,9 +15,33 @@ parser.add_argument('source', metavar='SOURCE-SHAPEFILE-PATH')
 parser.add_argument('boundary', metavar='NAME-OF_BOUNDARY')
 
 args = parser.parse_args()
+boundary_path = Path('./boundaries/build') / Path(args.boundary)
+source_path = Path(args.source)
 
+# Create a folder
 try:
-    boundary_path = Path('./boundaries/build') / Path(args.boundary)
     boundary_path.mkdir()
 except FileExistsError as e:
     sys.exit('{} already exists. Please pick a new name'.format(args.boundary))
+
+# Get existing json entries
+with open('./boundaries/boundary-config.json', 'r') as jsonFile:
+    try:
+        config_entries = json.load(jsonFile)
+    except json.decoder.JSONDecodeError as e:
```

Comment:
I think this is potentially a bit confusing - I think you've added this so that if there's a zero-byte JSON file there, it's treated as a JSON file of `[]`. But this will also catch any syntax error in the JSON, and treat it as if there's nothing in the file without producing an error, and actually overwrite the file with a syntax error losing anything that was previously in it.

I think we should assume that if the `.json` file exists at all, it should be well-formed JSON, and otherwise produce an error. (This will mean changing your test for this behaviour so that it doesn't `.touch()` the file.)

You can still handle the file being missing as "no config data" with something like the following:

```python
try:
    with open('boundaries/boundary-config.json') as f:
        config_entries = json.load(f)
except FileNotFoundError:
    config_entries = []
```